examples/generic.toml

```diff
@@ -0,0 +1,5 @@
+name = "foo"
+
+[address]
+    addressStreet = "Bar"
+    addressHouse = 42
```

Comment:
I guess you can create an issue to start thinking about a possible interface for improving DerivingVia experience and stripping options :slightly_smiling_face: